go-apps/meep-platform-ctrl/api/swagger.yaml

```diff
@@ -25,6 +25,21 @@ produces:
 - "application/json"
 paths:
   /login:
```

Comment:
Not a huge thing 
- would have expected an endpoint of its own for the watchdog (e.g. /watchdog) and the user to POSTing to that endpoint (e.g. user kicking the watchdog)
- whether the "kicking" process generates a new cookie or not is internal to the platform and should not necessarily be exposed through the API

Reply:
I also considered POSTing a heartbeat to a separate endpoint as an implementation option, therefore I will make the API modification as suggested.